SETUP.md

```diff
@@ -0,0 +1,12 @@
+# Setup Your Workspace.
+
+We're using Python 3 for this development (unless we want to support Python 2.7 as well?), which is idetified by the python3 calls below.
```

Comment:
I don't think at this point python 2.7 is something that I am worried about supporting.